src/Microsoft.AspNetCore.Authorization/AuthorizationOptions.cs

```diff
@@ -11,7 +11,10 @@ namespace Microsoft.AspNetCore.Authorization
     /// </summary>
     public class AuthorizationOptions
     {
-        private IDictionary<string, AuthorizationPolicy> PolicyMap { get; } = new Dictionary<string, AuthorizationPolicy>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Note: the equivalent PolicyMap in AuthenticationOptions today is public, so whatever we decide to do here, we should probably be consistent: https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationOptions.cs#L22